introduction.md

```diff
@@ -1,2 +1,5 @@
 Luke, I'm ma-ta!
 Favorite path: JavaScript
+Skills:
+* Git
+* JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?